src/backend/blockchain/src/routes.py

```diff
@@ -6,11 +6,40 @@
 from flask import request, jsonify
 import sys
 import os
+import json
+from src import peers
 
 sys.path.append(os.path.abspath(os.path.join('..', '')))
 
 from shared.exceptions import IncorrectPayloadException
 from shared import HttpCode
+from shared.utils import send_post_request, send_get_request
+
+request_handlers = {
+    "GET": send_get_request,
+    "POST": send_post_request
+}
+
+def query_peer(peers):
+    try:
+        backup_node = peers[0]
```

Comment:
I understand we only have a single peer as is but this is a bit tech-debty. Alternatively, I propose we use a loop instead to send requests to all the peer nodes.

Reply:
This is where we tread along the lines of consensus haha, but yeah a loop here will do because we only ever have one backup in this case

Reply:
Just one concern, the code here is just taking the chain from the backup node and is copying the entire chain and overriding the chain in the master node, `for` loop doesn't make sense too, because we will iterate over the peers and keep the chain from the last peer. Is there a better way to do it? I was thinking of storing the backup node's position in the list as a variable and using that instead of hardcoding `0`

Reply:
Ahhhh I see, I misread this, at first glance I thought this was where the peer was forwarding its chain copy to its known peers. Yes, this definitely treading close to consensus... Yeah, the only practical use of a loop here is if we threw in longest chain logic or something along those lines in here for dealing with multiple copies but we can assume the single backup as you have here. Resolve this after reading